user/example-models/simple_model.py

```diff
@@ -0,0 +1,20 @@
+# USER: Other framework (K1st) and service layer
+
+# WHAT:
+# Simple, understandable in less than 5 seconds for the term
+# Stressing on Knowledge
+
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__),'../../'))
```

Comment:
This should not be necessary in the actual code base, once you have the proper package.

Reply:
I have removed this part completely